languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/words_no_infix_s.txt

```diff
@@ -1418,4 +1414,183 @@ Zentrifugen
 Zionisten
 Zulagen
 Zutaten
-Zyklonen
\ No newline at end of file
+Zyklonen
+Abstrusitäten
+Achsen
+Adiabaten
+Agaven
+Ampullen
+Anekdoten
+Anleihen
+Anschriften
+Assistenten
+Astronomen
+Azteken
+Banderolen
+Basen
+Bauklötzchen
+Baumwollhemden
+Besen
+Biologen
+Chrysanthemen
+Debatten
+Debitoren
+Direktoren
+Direktorinnen
+Diätassistenten
+Drahtspeichen
+Dreiwelten
+Dänen
+Elektroden
+Enklaven
+E-Junioren
+F-Junioren
+G-Junioren
+D-Junioren
+C-Junioren
+B-Junioren
+A-Junioren
+Elbauen
+Espen
+Exoten
+Fabrikanten
+Familienbeihilfen
+Fanfaren
+Feen
+Fladen
+Flaschen
+Flächen
+Folien
+Furchen
+Fältchen
+Führergehilfen
+Gallen
+Gattinnen
+Gazellen
+Gemeinwesen
+Gesundbrunnen
+Giganten
+Girlanden
+Haxen
+Heiden
+Heinzelmännchen
+Herden
+Hemden
+Hitparaden
+Hymnen
+Häkchen
+Härchen
+Hündinnen
+Hütten
+Immigranten
+Inspektoren
+Interpreten
+Irokesen
+Jungsozialisten
+Junioren
+Juroren
+Kadetten
+Kameraden
+Kapitalisten
+Karaffen
+Kardiologen
+Karikaturen
+Katakomben
+Kathedralen
+Kathoden
+Kellnerinnen
+Kichererbsen
+Kieferorthopäden
+Klötzchen
+Knochen
+Kolonisten
+Kommandanten
+Kommunen
+Komplizen
+Konfitüren
+Kreditwesen
+Lawinen
+Lektoren
+Leserinnen
+Lianen
+Lobbyisten
+Linguisten
+Lotus
+Magmen
+Makronen
+Maiglöckchen
+Mattscheiben
+Maschinen
+Massen
+Memoiren
+Menschen
+Millisekunden
+Minoritäten
+Monarchen
+Mängelrügen
+Märchen
+Narzissen
+Neffen
+Nissen
+Nockenwellen
+Nymphen
+Obelisken
+Obergrenzen
+Oblaten
+Oligarchen
+Ordnungswidrigkeiten
+Paradigmen
+Patriarchen
+Pforten
+Pfötchen
+Phrasen
+Pillen
+Pinzetten
+Planken
+Plomben
+Pointen
+Polypen
+Primarstufen
+Prostituierten
+Protestanten
+Psychologen
+Pünktchen
+Raumfähren
+Reliquien
+Renten
+Repräsentanten
+Rouladen
+Salafisten
+Schafgarben
+Sandburgen
+Schamanen
+Schildkröten
+Schwefelsäuren
+Sensen
+Silhouetten
+Sozialisten
+Spatzen
+Spielchen
+Sportkameraden
+Schulkameraden
+Stalaktiten
+Stechpalmen
+Sternschnuppen
+Straßen
+Studenten
+Suchmaschinens
+Synagogen
+Tavernen
+Tiefkühltruhen
+Transvestiten
+Trauben
```

Comment:
Maybe add Tauben/Trauben to `ProhibitedCompoundRule`?

Reply:
https://github.com/languagetool-org/languagetool/pull/9638